torch/distributed/_state_dict_utils.py

```diff
@@ -29,6 +32,10 @@
     from torch.distributed._tensor import distribute_tensor, DTensor, Replicate
 
 
+logger = getLogger()
```

Comment:
are we using this logger?

Reply:
aah sorry, will clean this up - used for local debugging.